builtin/logical/ssh/secret_ssh_cert.go

```diff
@@ -0,0 +1,33 @@
+package ssh
```

Comment:
Since we don't have CRLs, there is nothing the backend can do when leases expire. This file can be deleted and a normal `logical.Response` can be returned from https://github.com/hashicorp/vault/pull/2208/files#diff-ee2dd305479a290de229124276a4aa0fR195.